4_arm.ipynb

```diff
@@ -23,11 +41,33 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Motion generation using forward kinematics\n",
+    "\n",
+    "\"Forward kinematics\" is to find the position and orientation of the robot's hand from the displacement of the joints. Let's move the arm of HSR by specifying the joint angles."
```

Comment:
"\"Forward kinematics\" is used to find the position and orientation of the robot's hand from the displacement of the joints. Let's move the arm of the HSR by specifying the joint angles."